web_client/views/body/ImageView.js

```diff
@@ -56,6 +56,9 @@ var ImageView = View.extend({
         this.listenTo(this.annotationSelector, 'h:deleteAnnotation', this._deleteAnnotation);
 
         this.listenTo(events, 's:widgetChanged:region', this.widgetRegion);
+        this.listenTo(events, 'g:login g:logout.success g:logout.error', () => {
+            this._openId = null;
```

Comment:
This is an improvement over the current (broken) behavior.  There is still an issue with the state of the panels remaining after logout/login.  I think the panels should be reset.  We can save that for a future PR though.